brains/openweathermap.yml

```diff
@@ -1,13 +1,14 @@
 ---
   - name: "get-the-weather"
     signals:
-      - order: "quel temps fait-il"
+      - order: "quel temps fait-il {{ location }}"
```

Comment:
we could keep both example with parameter from order or parameter into the neuron, doesn't it?